File: src/Neuralm.Domain/Entities/NEAT/Nodes/HiddenNode.cs
namespace Neuralm.Domain.Entities.NEAT.Nodes
{
    /// <summary>
    /// Represents the <see cref="HiddenNode"/> class.
    /// </summary>
    public class HiddenNode : Node
    {
        /// <summary>
        /// Initializes an instance of the <see cref="HiddenNode"/> class.
        /// </summary>
        /// <param name="nodeIdentifier"></param>
        public HiddenNode(uint nodeIdentifier) : base(nodeIdentifier) { }
    }
}

File: src/Neuralm.Presentation.Web/src/models/HiddenNode.ts
import Node from './Node';

/**
 * Represents the hidden node class.
 */
export default class HiddenNode extends Node {
  /**
   * Initializes a new instance of the HiddenNode class.
   */
  constructor(id: string, nodeIdentifier: number) {
    super(id, nodeIdentifier);
  }
}
